src/features/add-task/ui.tsx

```diff
@@ -14,6 +14,8 @@ export const AddTask = () => {
         (e: SyntheticEvent) => {
             e.preventDefault();
 
+            if (!taskName) return false;
```

Comment:
it's redundant. You need to disable `submit` button when the `taskName` is empty